woocommerce/woocommerce-ios
# Dynamic Dashboard: Separate top performers from store stat section

Part of #12487 


## Description

This PR separates the existing `storeStatsAndTopPerformers` dashboard card type to separate types `peformance` and `topPerformers`. A new view for the Top Performers section on the Dashboard screen has also been created - due to the large number of changes, the implementation for the top performer list will be added in a separate PR.

Note: The same config is being used for the selected time range for the store performance and top performer cards. We will need to separate the configs in a separate PR.

## Testing instructions

- Enable the feature flag `dynamicDashboard` and build the app.
- Log in to a store and confirm that the Top Performers section should be displayed if it's enabled.
- Switch between different time ranges, the time range text should be updated accordingly.
- Tap on View all store analytics button, the analytics hub should be displayed.

## Screenshots


<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/ec38c825-f856-4b5c-a454-c1d476fcc67d" width=320 />

---
